### ISSUE TITLE
[request] Add file logging

### ISSUE THREAD

**Issue by gotspatel** [crazy-max/geoip-updater#180]
### Description

can you add file logging and log rotation if possible

**Comment by crazy-max** [comment]
Yes I think we could implement this but not just for Windows. It should be part of the logging logic.
